src/index.css

```diff
@@ -1,76 +1,353 @@
 body {
-  background-color: rgb(250, 250, 250);
+  margin: 0;
 }
 
 * {
+  margin: 0;
   padding: 0;
+}
+
```

Comment:
Please be sure that your widths are working properly. Im testing it with a width of 1024px and the result is:

#### Your project 
![Captura](https://user-images.githubusercontent.com/15807343/185690960-a33361a2-8bfd-4b13-ac66-715809eb7d4d.PNG)

Also, kindly fix the white background for the form element. As you can see, the color stops at the end of the button and it should keep going as shown in the design 

#### Your project 
![image](https://user-images.githubusercontent.com/15807343/185691255-52171546-5152-4d96-b0db-1445678db9ee.png)

#### Given Design  
![image](https://user-images.githubusercontent.com/15807343/185691436-bb5851d1-d61a-45fd-b7eb-d23cf605dab9.png)